src/kixi/hecuba/onyx/weather.clj

```diff
@@ -49,7 +55,7 @@
        (clojure.string/join \newline errors)))
 
 (defn exit [status msg]
-  (println msg)
+  (timbre/info msg)
   (System/exit status))
 
 (defn assert-job-exists [job-name]
```

Comment:
No, but the exit function is used in a couple of other places so I'd rather leave it in.